.evergreen/auth_oidc/azure/create-and-setup-vm.sh

```diff
@@ -10,9 +10,11 @@ if [ -z "${AZUREOIDC_VMNAME_PREFIX:-}" ]; then
     exit 1
 fi
 
+DIR=$(dirname ${BASH_SOURCE:-$0})
+. $DIR/../../handle-paths.sh
+
 # Set defaults.
-AZUREOIDC_DRIVERS_TOOLS=${AZUREOIDC_DRIVERS_TOOLS:-$DRIVERS_TOOLS}
```

Comment:
Was there a reason why a separate variable was introduced here? I don't think it is necessary, just doing my due diligence here.